.eslintrc.cjs

```diff
@@ -3,7 +3,7 @@ module.exports = {
     "no-console": "error",
   },
   parserOptions: {
-    ecmaVersion: 2020,
+    ecmaVersion: 2021,
```

Comment:
This is done because I used [numeric separator](https://2ality.com/2018/02/numeric-separators.html) in code. `ecmaVersion` 2020 doesn't support them. They are supported in Node ^12.5.0 so we should be good ([more details](https://github.com/nodejs/node/issues/28427#issuecomment-505588628)).